tests/integration/compose/docker_compose_local_kms.yml

```diff
@@ -0,0 +1,7 @@
+version: '2.3'
+services:
+    local_kms:
+        image: nsmithuk/local-kms:3.12.0
+        restart: always
+        ports:
+            - 4599:8080
```

Comment:
Internally Local KMS it's using the default port 8080. I exposed the 4599 so that we could ping the server from the outside for testing purposes because it's the one always shown [in their doc](https://github.com/nsmithuk/local-kms?tab=readme-ov-file#using-lkms-with-the-cli). However, I think it's better if I just export the same port that it's used and that port is one from the `PortPoolManager` as we do for Kafka :+1: